docs/getting-started/your-first-set-of-documentation.rst

```diff
@@ -96,22 +96,22 @@ DocBlocks are divided into the following three parts. Each of these parts is opt
 may not exist without a Summary.
 
 Summary
-  Sometimes called a short description, provides a brief introduction into the function of the associated element.
-  A Summary ends
-  in one of these situations:
+  The Summary, sometimes called a short description, provides a brief introduction into the function
+  of the associated element. A Summary ends when it encounters either of the below situations::
```

Comment:
I noticed the double colon here earlier, but assumed it was formatting... is it actually just a duplicate?

Reply:
The colon has special meaning in the markup language used, so for a literal colon it needs to be doubled in most cases.

Reply:
I wasn't looking for that previously... does it need to be doubled above on line 82 also?

Reply:
Looking at the generated docs, that one seems to be fine, though doubling it probably doesn't do any harm either.

Reply:
The double colon actually means: the next quoted or indented block is a Literal Block :) I think a slash is the one for escaping and I am not even sure that you need to escape a single colon